app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package io.pumpkinz.pumpkinreader.model;
+
+import java.util.Date;
+
+
+public abstract class Item {
```

Comment:
Hemm, but maybe I can tap the Retrofit return value and instantiate the object manually. I'll take a look.